presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -119,32 +128,17 @@ fun EventScreen(
                     )
 
                     Spacer(modifier = modifier.height(26.dp))
-
-                     AgendaDropDownMenu(
-                        modifier = Modifier
-                            .background(color = MaterialTheme.colorScheme.dropDownMenuBackgroundColor),
-                        shouldOpenDropdown = eventScreenState.shouldOpenDropdown,
-                        onCloseDropdown = {
-                            eventScreenEvent(
-                                EventScreenEvent.OnShowAlarmReminderDropdown(shouldOpen = false))
-                        },
-                        listOfMenuItemId = AlarmReminderItem.values().map { alarmReminderItem ->
-                            alarmReminderItem.stringResId
-                        },
-                        onSelectedOption = { item ->
-                            eventScreenEvent(EventScreenEvent.OnAlarmReminderChanged(AlarmReminderItem.values()[item]))
-                            eventScreenEvent(EventScreenEvent.OnShowAlarmReminderDropdown(shouldOpen = false))
-                        }
-                    )
-
+                    
                     AlarmReminder(
                         reminderText = stringResource(id = eventScreenState.alarmReminderItem.stringResId),
                         modifier = Modifier
                             .fillMaxWidth()
                             .background(color = MaterialTheme.colorScheme.backgroundWhiteColor),
                         isEditMode = eventScreenState.isEditMode,
                         onReminderClicked = {
-                            eventScreenEvent(EventScreenEvent.OnShowAlarmReminderDropdown(shouldOpen = true))
+                            bottomSheetScope.launch {
+                                bottomSheetState.show()
+                            }
```

Comment:
I am using the coroutineScope to show and hide the bottom sheet. But I am wondering if I should be calling an event in my EventScreenEvent class and keeping the state in my EventScreenState like I was doing for showing and hide the DropdownMenu? Or does this logic all belong in this EventScreen?

Reply:
It's fine to have it here